src/components/BaseWizard/hooks/useWizardProps.js

```diff
@@ -0,0 +1,86 @@
+import { computed, ref } from '@vue/composition-api';
+import getProgressStepperOptions from '../utils/getProgressStepperOptions';
+
+const props = ref();
+
+const isLoading = computed(() => props.value.isLoading);
+
+const isFirstStep = computed(() => props.value.isFirstStep);
+const isLastStep = computed(() => props.value.isLastStep);
+
+const fullScreen = computed(() => props.value.fullScreen);
+
+const showFooter = computed(() => props.value.showFooter);
+
+const showProgress = computed(() => props.value.showProgress);
+
+const title = computed(() => props.value.title);
+
+const isComplete = computed(() => props.value.isComplete);
+
+const compact = computed(() => props.value.compact);
+
+const currentLayout = computed(() => props.value.layout);
+const mainContent = ref(null);
+const currentStep = computed(() => props.value.currentStep);
```

Comment:
Not opposed to this if this is the best way of handling it. I almost feel like there's gotta be a better way. Almost like a computed version of `toRefs`? Or just a single computed for all refs? Just thinking out loud. Not requesting any change. 

Maybe we could also remove white space, if we're sticking with this method. 

```suggestion
const isLoading = computed(() => props.value.isLoading);
const isFirstStep = computed(() => props.value.isFirstStep);
const isLastStep = computed(() => props.value.isLastStep);
const fullScreen = computed(() => props.value.fullScreen);
const showFooter = computed(() => props.value.showFooter);
const showProgress = computed(() => props.value.showProgress);
const title = computed(() => props.value.title);
const isComplete = computed(() => props.value.isComplete);
const compact = computed(() => props.value.compact);
const currentLayout = computed(() => props.value.layout);
const mainContent = ref(null);
const currentStep = computed(() => props.value.currentStep);
```